lib/fastly_nsq/manager.rb

```diff
@@ -24,7 +24,7 @@ def initialize(logger: FastlyNsq.logger, **pool_options)
     @done      = false
     @logger    = logger
     @pool      = FastlyNsq::PriorityThreadPool.new(
-      { fallback_policy: :caller_runs, max_threads: DEFAULT_POOL_SIZE }.merge(pool_options),
+      { fallback_policy: :caller_runs, max_threads: FastlyNsq.max_threads || DEFAULT_POOL_SIZE }.merge(pool_options),
```

Comment:
I think maybe a little cleaner if you made this a named parameter in the method signature and defaulted `FastlyNsq.max_threads` to `DEFAULT_POOL_SIZE`

Reply:
a named parameter for which method?

Reply:
`#initialize`

Reply:
oh. I didn't want to mess with that since it's just doing

```
def initialize(*)
  super
  
  @queue = FastlyNsq::PriorityQueue.new(:max)
end
```

Reply:
oh sorry, I meant `FastlyNsq::Manager#initialize(logger: FastlyNsq.logger, **pool_options)` so

```ruby
class FastlyNsq::Manager
  def initialize(logger: FastlyNsq.logger, max_threads: FastlyNsq.max_threads, **pool_options)
	...
  end
end
```

Reply:
ohhh. that makes a lot more sense